packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -544,13 +544,16 @@ class AnimationController extends Animation<double>
   ///
   /// Defaults to repeating between the lower and upper bounds.
   ///
+  /// With [reverse] set to true, instead of always starting over at the lower
+  /// bound the value will alternate between the lower and upper bounds on each repeat.
```

Comment:
Need to rephrase this in terms of [min] and [max].

Reply:
Also did that 👍